src/internal/m365/collection/drive/limiter_test.go

```diff
@@ -880,151 +771,225 @@ func (suite *LimiterUnitSuite) TestGet_PreviewLimits_Defaults() {
 				MaxItemsPerContainer: 99999999,
 				MaxPages:             99999999,
 			},
-			expect: expected{
+			expect: defaultLimitTestExpects{
 				numItems:             int(defaultPreviewMaxBytes) / 1024 / 1024,
 				numContainers:        1,
 				numItemsPerContainer: int(defaultPreviewMaxBytes) / 1024 / 1024,
 			},
 		},
 	}
-	for _, test := range table {
+}
+
+// TestGet_PreviewLimits_Defaults checks that default values are used when
+// making a preview backup if the user didn't provide some options.
+// These tests run a reduced set of checks that really just look for item counts
+// and such. Other tests are expected to provide more comprehensive checks.
+func (suite *LimiterUnitSuite) TestGet_PreviewLimits_Defaults_noTree() {
+	// Add a check that will fail if we make the default smaller than expected.
+	require.LessOrEqual(
+		suite.T(),
+		int64(1024*1024),
+		defaultPreviewMaxBytes,
+		"default number of bytes changed; DefaultNumBytes test case may need updating!")
+	require.Zero(
+		suite.T(),
+		defaultPreviewMaxBytes%(1024*1024),
+		"default number of bytes isn't divisible by 1MB; DefaultNumBytes test case may need updating!")
+
+	metadataPath, err := path.BuildMetadata(
+		tenant,
+		user,
+		path.OneDriveService,
+		path.FilesCategory,
+		false)
+	require.NoError(suite.T(), err, "making metadata path", clues.ToCore(err))
+
+	drive1 := models.NewDrive()
+	drive1.SetId(ptr.To(idx(drive, 1)))
+	drive1.SetName(ptr.To(namex(drive, 1)))
```

Comment:
nit: all of this could be done in `runGetPreviewLimitsDefaults`. Could you please move it there instead of having it in each separate top-level test function?

Reply:
done